documentation/book/proc-replacing-private-keys.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-replacing-private-keys-{context}']
+
+= Replacing private keys
+
+You can replace the private keys used by the cluster CA and clients CA certificates. When a private key is replaced, the Cluster Operator generates a new CA certificate for the private key.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A Kafka cluster in which CA certificates and private keys are installed.
+
+.Procedure
+
+. View a description of the private key `Secret` that contains the private key that you want to renew.
+
+.. To view the `Secret` for the cluster CA certificate's private key: 
++ 
+[source,shell,subs="+quotes"]
+kubectl describe secret _<cluster-name>_-cluster-ca
+
+.. To view the `Secret` for the clients CA certificate's private key:
```

Comment:
...the client's CA certificate's...? If so, I might reword to `To view the `Secret` for the private key of the client's CA certificate`

Reply:
Yes, there is some potential ambiguity. I'll reword.

Reply:
There are two CA certificates: the "cluster CA certificate" and the "clients CA certificate". I've reworded as follows:

To view the `Secret` for the private key for the cluster CA certificate.

 To view the `Secret` for the private key for the clients CA certificate.